README.md

```diff
@@ -41,7 +41,7 @@ To install agent5, you need to:
 - either set `datadog_agent_version` to an existing agent5 version
   (recommended) or leave it empty to always install the latest version (`5.*`).
 
-To downgrade from agent6 to agent5, you need to:
+To downgrade from agent6 to agent5, you need to (**on centos this will only work with ansible 2.4+**):
```

Comment:
nit: `ansible 2.4 and up`?

Reply:
you should also update the description of the `datadog_agent_allow_downgrade` variable in the `Role Variables` section of the readme